CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [2.31.0] - 2022-03-21
+## [2.31.0] - 2022-06-06
```

Comment:
You do not need to change this date. It specifies when 2.31.0 development started.